tests/feats/feature.py

```diff
@@ -189,6 +189,24 @@ def reticulate_string(self, value: str) -> int:
             return 0
 
 
+class InvalidBooleanFeatureAsClass:
```

Comment:
Let's add a test for the @app.feature and @app.segment that they cannot be plain functions.